pkg/util/resource_windows.go

```diff
@@ -0,0 +1,16 @@
+package util
+
+import (
+	"fmt"
+
+	"github.com/docker/go-units"
+)
+
+func ParseUlimit(ulimit string) (*units.Ulimit, error) {
```

Comment:
It worries me a bit that there is a non-Linux function.  Is this being called by the remote client?  I think it only really makes sense to evaluate `-1` on the server side.

Can you make sure that's the case?

Reply:
This is a buildah PR not a Podman PR (Yet, when I move this on the Podman side, I will update the tests).

Reply:
Thanks!  I didn't notice it was a Buildah PR :see_no_evil: